Makefile

```diff
@@ -2,6 +2,7 @@ GOBASE=$(shell pwd)
 GOBIN=$(GOBASE)/bin
 GO_BUILD_FLAGS := ${GO_BUILD_FLAGS}
 ROOT_DIR := $(or ${ROOT_DIR},$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST)))))
+TIMEOUT ?= 30m
```

Comment:
I think we can reduce this :)

Reply:
hehe may be 10min for now would be enough. We can bump it later if needed.

But I think it's just more important to limit specific tests that wait to not wait more than expectable.